elements/rh-stat/rh-stat.css

```diff
@@ -38,6 +36,10 @@ pfe-icon[size="md"],
   display: block;
   color: black;
```

Comment:
Looks like if we remove the color property for icon it will inherit the light / dark colors based on the theme.

Reply:
This seems preferable to adding a `:host([on="dark"])` property. Would there be a case where a RH web property would have a different default icon color though?

Reply:
The reason for the behaviour @brianferry observed was because rh-stat was using the colour context from pfe, which is slated to be deprecated and removed.

The last few commits to this branch replace the pfe context with the correct rhds context, and demonstrate how to remove the `on` attribute in favour of a private shadow class, as discussed in #625